### PR TITLE
feature/empty splash screen

### LINKED ISSUE #19: [UI] Add Empty Splash Screen

### PR DESCRIPTION
**Pull request by keithchad**
I added splash.view package then added a Kotlin class for the Splash screen. I then added it to Screens then I changed the first screen to Splash screen and added a two second delay then it navigates to Home screen.
Closes #19

### CODE REVIEW

**Review by chepsi — CHANGES_REQUESTED** [review]
Can we use [splash api](https://developer.android.com/guide/topics/ui/splash-screen) instead.

**Comment by keithchad** [comment]
> Can we use [splash api](https://developer.android.com/guide/topics/ui/splash-screen) instead.

Yea sure. Working on it.

**Comment by keithchad** [comment]
> Can we use [splash api](https://developer.android.com/guide/topics/ui/splash-screen) instead.

I added the [splash api](https://developer.android.com/guide/topics/ui/splash-screen) dependency to presentation Gradle file then I then I created a theme for the Splash screen then I added a drawable icon for the animated icon then I called installSplashScreen in MainActivity.kt

**Review by chepsi — APPROVED** [review]
LGTM

File: presentation/build.gradle.kts [hunk 1]
```diff
@@ -64,6 +64,7 @@ dependencies {
     implementation(libs.bundles.compose)
     implementation(libs.lifecycle.runtimeKtx)
     implementation(libs.android.hilt)
+    implementation("androidx.core:core-splashscreen:1.0.0-rc01")
```
Comment (jumaallan) [inline presentation/build.gradle.kts:67]: hey @keithchad Can you define this dependency like the others

Basically:
1. Look for `libs.versions.toml`, and define this dependency there (top of the file [versions] defines the version, then add it under the [libraries] section) - see how other dependencies are setup
2. Then use it on the presentation build file like ` implementation(libs.androix.splashscreen)`

**Review by jumaallan — CHANGES_REQUESTED** [review]
Left some small review

File: presentation/build.gradle.kts (hunk 1 above)
Comment (keithchad) [reply presentation/build.gradle.kts:67]: Okay. Working on it.

Comment (keithchad) [reply presentation/build.gradle.kts]: I have updated both Gradle files with the right implementation.

Comment (jumaallan) [reply presentation/build.gradle.kts]: Thank you

**Review by jumaallan — APPROVED** [review]

**Pull request merged**